test/OpenTelemetry.Exporter.Geneva.Tests/GenevaTraceExporterTests.cs

```diff
@@ -359,7 +359,7 @@ public void GenevaTraceExporter_Constructor_Missing_Agent_Linux()
                         options.ConnectionString = "Endpoint=unix:" + path;
                     })
                     .Build();
-                Assert.True(false, "Should never reach here. GenevaTraceExporter should fail in constructor.");
+                Assert.True(true, "GenevaTraceExporter should not fail in constructor.");
```

Comment:
Could you add a comment here saying "GenevaExporter would not throw if it was not able to connect to the UDS socket in ctor. It would keep attempting to connect to the socket when sending telemetry."